src/assets/scss/_global.scss

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'gibsonlight';
```

Comment:
I _believe_ I spoke to Chris about this last time and he decided it would be better to have consistency and use Helvetica/Arial. Probs better to check with him again though.